web/src/components/NavMenu.js

```diff
@@ -1,17 +1,36 @@
 /* eslint react/prop-types: 0 */
 import { React, useState } from "react";
-import IconButton from "@material-ui/core/IconButton";
+import { Button, IconButton } from "@material-ui/core";
 import Menu from "@material-ui/core/Menu";
 import MenuItem from "@material-ui/core/MenuItem";
 import MenuIcon from "@material-ui/icons/Menu";
 //import AuthService from "../services/auth.service";
+import WalletService from "../services/wallet.service";
 import { useHistory } from "react-router-dom";
 
 const NavMenu = (props) => {
   const { onLogout } = props;
   const [anchorEl, setAnchorEl] = useState(null);
   const history = useHistory();
 
+  const [wallet, setWallet] = useState(WalletService.checkAccount);
+  WalletService.changeAccounts(setWallet);
+
+  const connect_button =
+    wallet == null ? (
+      <Button
+        variant="contained"
+        color="secondary"
+        onClick={() => WalletService.connectToWallet()}
+      >
+        Connect Wallet
+      </Button>
+    ) : (
+      <Button variant="contained" color="secondary" disabled>
+        Wallet Connected
```

Comment:
I think the text is too big, maybe just "Connected"?

Reply:
Done